internal/factsengine/gatherers/sbd_test.go

```diff
@@ -17,6 +17,14 @@ func TestSBDGathererTestSuite(t *testing.T) {
 	suite.Run(t, new(SBDGathererTestSuite))
 }
 
+func (suite *SBDGathererTestSuite) TestDefaultSBDConfigGagherer() {
```

Comment:
As agreed offline, let's remove this test. We accept the coverage reduction in this case, as in fact, the reduction simply comes for the fact that there are less lines of code. This function was not tested beforehand neither

Reply:
+1, I lifted the threshold to 1%

Reply:
Alright, thanks!